modules/essentials/metrics_server.tf

```diff
@@ -0,0 +1,13 @@
+locals {
+  helm_config = merge(
+    var.metrics_server_helm_config_defaults,
+    var.metrics_server_helm_config
+  )
+}
+
+module "helm_metrics_server" {
+  count  = var.metrics_server_enabled ? 1 : 0
+  source = "github.com/SPHTech-Platform/terraform-helm-release?ref=init"
```

Comment:
Once the module PR is approved and merged, this will be updated to registry url.